src/main/java/Duke.java

```diff
@@ -1,59 +1,122 @@
 import java.util.Scanner;
+import java.util.Arrays;
 import java.util.ArrayList;
 
 public class Duke {
 
     public static String indent = "     ";
 
+    // init array of Task objects
     public static ArrayList<Task> allTasks = new ArrayList<>();
 
+    // helper function to convert string array to string
+    // not just string rep like what Arrays.toString(strArr) does
+    public static String convertToString(String[] strArr) {
+//        StringBuilder builder = new StringBuilder();
+//        for(String s : strArr) {
+//            builder.append(s + " ");
+//        }
+//        return builder.toString();
+        return String.join(" ", strArr);
+    }
+
+    // print array of Task objects
     public static void printAllTasks() {
         for (int i = 0; i < allTasks.size(); i++) {
-            String symbol = allTasks.get(i).getIsDone() ? "\u2713" : "\u2717" ;
-            System.out.println(indent + (i + 1) + ". [" + symbol + "] " + allTasks.get(i).getTask());
+            System.out.println(indent + (i + 1) + ". " + allTasks.get(i));
         }
     }
 
-    public static void updateTaskAsDone(int num) {
-        if (num >= allTasks.size()) {
+    // print newly added task
+    public static void printNewlyAddedTask(Task task) {
+        System.out.println(indent + "Got it. I've added this task: "
+                + System.lineSeparator()
+                + indent + indent + task
+                + System.lineSeparator()
+                + indent + "Now you have " + allTasks.size() + " tasks in the list.");
+    }
+
+    // mark a Task as done
+    public static void markTaskAsDone(int num) {
+        if (num > allTasks.size() || num == 0) {
             System.out.println(indent + "Sorry, this task does not exist!");
         }
         else {
```

Comment:
Should this 'else' be on the same line as the closing brace of the 'if' statement?